src/index.html

```diff
@@ -9,6 +9,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div>
+        <a href="index.html">
+          <img
+            class="logo"
+            src="images/logo.png"
+            alt="logo"
+          >
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li>
+            <a class="nav__link is-active" href="index.html">APPLE</a>
```

Comment:
do not write text in uppercase from scratch - use CSS `text-transform: uppercase;` instead